src/Tribe/Aggregator/Record/Meetup.php

```diff
@@ -77,4 +77,55 @@ public static function filter_event_to_preserve_fields( $event, $record ) {
 
 		return self::preserve_event_option_fields( $event );
 	}
+
+	/**
+	 * Returns the Meetup authorization token generation URL.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args
+	 *
+	 * @return string Either the URL to obtain Eventbrite authorization token or an empty string.
+	 */
+	public static function get_auth_url( $args = array() ) {
+		$service = tribe( 'events-aggregator.service' );
+
+		if ( $service->api() instanceof WP_Error ) {
```

Comment:
Minor tweak, but I suggest we set `$api` above this line and reference that here, and remove it from being set further below.

Reply:
Yep, better, thanks.